HW_3/task18.py

```diff
@@ -0,0 +1,25 @@
+'''Задача 18: Требуется найти в массиве A самый близкий по величине элемент к заданному числу X. 
+Если таких элементов несколько, вы вывести один любой. Пользователь в первой строке вводит натуральное число
+N – количество элементов в массиве. В последующих  строках записаны N целых чисел Ai (можно использовать 
+псевдогенерацию). Последняя строка содержит число X
+*Пример:*
+5
+    7 -2 3 5 1
+    6
+    -> 7. '''
+
+
+import random 
+n = int(input("Введите количество элементов в массиве: "))
+a = [random.randint(1, 10) for i in range(n)]
+print("Массив A:", a)
+X = int(input('Введите число X, с которым необходимо сравнивать элементы списка: '))
+min = abs(X - a[0])
+index = [0]
```

Comment:
Ну так, если вы задали переменную `min` таким значениям, то это переменная должна быть такой:
```suggestion
index = a[0]
```